openslides/topics/static/js/topics/site.js

```diff
@@ -12,6 +12,9 @@ angular.module('OpenSlidesApp.topics.site', ['OpenSlidesApp.topics'])
                 url: '/topics',
                 abstract: true,
                 template: "<ui-view/>",
+                data: {
+                    title: 'Topics',
```

Comment:
"Topics" is not yet translated. Suggestion: Use already translated string "Topic" because the page title is only used for topic detail view (for list view we use the agenda app).